docs/source/training_tutorials/finetune_llm.mdx

```diff
@@ -16,6 +16,8 @@ limitations under the License.
 
 # Fine-tune and Test Llama-3 8B on AWS Trainium 
 
+[[open-in-collab]]
+
```

Comment:
```suggestion
```

We won't want to open it in collab don't we?

Reply:
You're right. We want to generate notebooks automatically but not open then in collab. Removed.